charts/geo-addressing/charts/addressing-svc/templates/deployment.yaml

```diff
@@ -45,17 +45,19 @@ spec:
             httpGet:
               path: /health
               port: http
-            initialDelaySeconds: 60
-            failureThreshold: 60
-            timeoutSeconds: 10
-            periodSeconds: 60
+            initialDelaySeconds: 10
```

Comment:
Is this similar to what we have in DIS?

Reply:
Yes, this is similar to what we have in DIS.